.github/workflows/typescript-publish.yml

```
@@ -50,25 +58,81 @@
+          echo "EOF" >> $GITHUB_OUTPUT
 
+      - name: Create GitHub Release
+        if: steps.version.outputs.has_changes == 'true'
+        uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        with:
+          tag_name: typescript-v${{ steps.version.outputs.new_version }}
+          release_name: TypeScript Package v${{ steps.version.outputs.new_version }}
+          body: |
```

actionlint (1.7.7)

property "changelog" is not defined in object type {changelog_text: {conclusion: string; outcome: string; outputs: {string => string}}; current_version: {conclusion: string; outcome: string; outputs: {string => string}}; version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)